### DISCUSSION TITLE
Mysql Version

### DISCUSSION THREAD

**Discussion by aravindrnair05** [Coral-erm/Coral#769] [Q&A]
Dear Team,

Please mention which version of MySQL database will support for the new coral 2024.04

**Comment by andyp-uk** [comment]
We are using MariaDB 10.11.7 without any problems.

**Comment by aravindrnair05** [comment]
Can you tell me whether i can use MySQL for Coral, if possible, can you tell me the version

**Comment by tarif20** [comment]
I am using Mariadb 10.6, php 8.3, Ubuntu 22.04 
still I am fetching same problems, please help me
![Screenshot from 2024-05-17 18-23-04](https://github.com/coral-erm/coral/assets/170079481/20d56638-6658-4329-8f1e-62bf506a8f84)
